### DISCUSSION TITLE
Unable to reproduce MDN results

### DISCUSSION THREAD

**Discussion by mbelalsh** [manujosephv/pytorch_tabular#481] [General]
Hello there. My question is in reference to mixture density network results on non-linear example [here](https://pytorch-tabular.readthedocs.io/en/latest/tutorials/07-Probabilistic%20Regression%20with%20MDN/#non-linear-example). I copied the code over to my local machine and ran it. The results are not what you have on the library page. I got the following results. 

![prob_reg_mdn_2](https://github.com/user-attachments/assets/c670559b-0b41-4d59-b10a-225567009b1c)

**Comment by manujosephv** [comment]
MDN results are highly stochastic. So, it stands to reason you not getting the same kind of output.
